### DISCUSSION TITLE
Does anyone know a working download link for Firefox Nightly (besides APKmirror and APKPure).

### DISCUSSION THREAD

**Discussion by DwainZwerg** [ImranR98/Obtainium#621] [Q&A] [answered]
Hello,
in GitHub I can't find [Firefox Nightly](https://github.com/mozilla-mobile/firefox-android/releases), I only find other Firefox versions.

**Comment by Word66** [comment, marked as answer]
https://ftp.mozilla.org/pub/fenix/nightly/

**Reply by DwainZwerg** [reply]
Thanks for your help.
Can you help me find the correct REGEX for the armeabi-v7a/ varaint?

**Reply by Word66** [reply]
You can also get it from here, but I don't see anything you can key off of for versioning.

https://firefox-ci-tc.services.mozilla.com/tasks/index/mobile.v3.firefox-android.apks.fenix-nightly.latest/armeabi-v7a

**Reply by ypxun** [reply]
Hi, Word66. Thanks for your link[](https://ftp.mozilla.org/pub/fenix/nightly/). But it didn't work for me. When I added this link on Obtainium application subscription page as type HTML, it told "Not Found".<br>
Can you share how you subscribe this link?

**Reply by Word66** [reply]
I'm using the beta through GitHub honestly. I'm just providing links to the nightly apps, no idea what the logic looks like.

**Reply by foolishgrunt** [reply]
Any chance you'd want to share your code for FireFox Beta (from Github) to https://apps.obtainium.imranr.dev/? I tries working it out myself, but RegEx is a foreign language to me.

**Comment by Mikaela** [comment]
I have decided it as too complicated and just have https://github.com/Tobi823/ffupdater in Obtainium instead.

**Reply by DwainZwerg** [reply]
I simply update the app from the AuroraStore. I need it anyway for the PayPal and the banking app, which I don't want to download from apkmirror.

**Comment by ghost** [comment]
Use this link (https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/mobile.v3.firefox-android.apks.fenix-nightly.latest.arm64-v8a/artifacts/public%2Fbuild%2Ftarget.arm64-v8a.apk) - and choose override at direct APK link.
It's worked but have a problem, this link is too old. I'm researching a new way to find this.

**Reply by DwainZwerg** [reply]
You can find the correct configuration for Firefox [here](https://apps.obtainium.imranr.dev/?categories=browser). Forgot to close this age-old discussion.
